packages/nbmodel/src/api.ts

```diff
@@ -316,7 +316,37 @@ export interface ISharedCodeCell
   /**
    * Execution, display, or stream outputs.
    */
-  getOutputs(): nbformat.IOutput[];
+  getOutputs(): Array<nbformat.IOutput>;
+
+  /**
+   * Add/Update output.
+   */
+  setOutputs(outputs: Array<nbformat.IOutput>): void;
+
+  /**
```

Comment:
Instead of having three different methods, we should follow the pattern of having a single method for doing these manipulations as we do with `updateSource(start: number, end: number, value?: string): void;`. I also prefer simple insert & delete methods, but for consistency with the modeldb interface we should use a similar implementation as `updateSource`.

Reply:
Do you mean for insert and delete output?

Reply:
Yeah exactly. there should only be `updateOutputs` and `setOutputs`, similarly to the implementation of `source` and `cells`.

Reply:
Cells have `getCell`, `insertCell`, `insertCells`, `moveCell`, `deleteCell` and `deleteCellRange`.

Reply:
Right. Maybe we should change that next.